src/Serilog.Sinks.OpenTelemetry/OpenTelemetryLoggerConfigurationExtensions.cs

```diff
@@ -85,4 +85,65 @@ public static class OpenTelemetryLoggerConfigurationExtensions
             options.Protocol = protocol;
         });
     }
+    
+    /// <summary>
+    /// Audit to an OTLP exporter, waiting for each event to be acknowledged, and propagating errors to the caller.
+    /// </summary>
+    /// <param name="loggerAuditSinkConfiguration">
+    /// The `AuditTo` configuration object.
+    /// </param>
+    /// <param name="configure">The configuration callback. Note that </param>
```

Comment:
Is there something missing from the description? Note that...

Reply:
Nice catch, this was left over from an earlier attempt to reuse the same "options" type across both methods. Fixed! 👍